drivers/tee/tee_core.c

```diff
@@ -241,6 +241,9 @@ static int params_from_user(struct tee_context *ctx, struct tee_param *params,
 			    struct tee_ioctl_param __user *uparams)
 {
 	size_t n;
+	struct tee_ioctl_version_data *vers;
+
+	tee_client_get_version(ctx, &vers);
```

Comment:
This is a bit heavy. How about adding a `bool cap_memref_null` which is initialized with the context?